api-report/container-runtime.api.md

```diff
@@ -657,6 +668,9 @@ export class PendingStateManager implements IDisposable {
     replayPendingStates(): void;
 }
 
+// @public
+export function requestSummarizer(loader: ILoader, lastSequenceNumber: number, { cache, reconnect, summarizingClient, url }: ISummarizerRequestOptions): Promise<ISummarizer>;
```

Comment:
(nit) It's a bit weird to see expansion of ISummarizerRequestOptions argument here. Not a big deal, but it would look nicer if you expand args in the body of the function, not in argument list.